README.rst

```diff
@@ -480,6 +502,35 @@ advantage is that the script support OS_TRUST_ID, while other tools as nova does
 
 Top_
 
+Acceptance tests with Docker execution
+**************************************
+Skuld acceptance tests can be executed by Docker. To do that, firtly it is required to create the docker image.
+To do that:
+
+.. code::
+
+    docker build -t fiware-skuld -f docker/Dockerfile docker
+
+Once the fiware-skuld image is created, we can run it by using docker-compose (to include the environment variables). To export then
+is required:
+
+.. code::
+
+    export OS_AUTH_URL = {the auth uri of the testbed agains the tests are going to be execute}
+    export OS_USERNAME = {the user name}
+    export OS_TENANT_NAME = {the tenant name}
+    export OS_PASSWORD = {the password}
+    export OS_REGION_NAME = {the region}
+    export OS_PROJECT_DOMAIN_NAME = {the project domain name}
+    export OS_USER_DOMAIN_NAME = {the user domain name}
+    docker-compose -f docker/docker-compose.yml up
+
+When docker has finished, you can obtain the tests results by
+.. code::
+
+   docker cp docker_fiwareskuld_1:/opt/fiware-skuld/tests/acceptance/testreport .
+
+Top_
```

Comment:
Yes, I think that you should explain more it, what is a testbed and where is it or how to create it.